src/test/java/String/SetCollectionTest.java

```diff
@@ -0,0 +1,44 @@
+package String;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
```

Comment:
이미 우리는 Set이 Collection API에서 제공하는 자료구조중 하나라는 것을 알고 있기에 Collection을 명시하지 않아도 되지 않을까요? 
```suggestion
public class SetTest {
```